autoload/lsp.vim

```diff
@@ -350,7 +350,11 @@ function! s:ensure_init(buf, server_name, cb) abort
     if has_key(l:server_info, 'capabilities')
         let l:capabilities = l:server_info['capabilities']
     else
-        let l:capabilities = {}
+        let l:capabilities = {
```

Comment:
@prabirshrestha is this the correct way to pass the client capabilities ?

Reply:
Try `applyEdit: v:true` instead